src/app/_components/Search.tsx

```diff
@@ -16,7 +16,7 @@ import { PAGE_KEY, SEARCH_KEY } from '@/constants/searchParams'
 
 export type SearchProps = {
   query: ReturnType<typeof useSearch>['searchQuery']
-  id: 'mobile-search' | 'web-search'
+  id: string
```

Comment:
I see. I reverted back to `string` because this `Search` component is used for both desktop and mobile, so neither of these options is accurate.

Do you see what I mean @barbaraperic? Do you have a better idea of how to make it work, maybe?